### PR TITLE
Add new Hosting PlanetHoster.net

### PR DESCRIPTION
**Pull request by NormandLemay**

### CODE REVIEW

**Comment by mxxcon** [comment]
@NormandLemay I think it's better suited for `Hosting/VPS` section rather than domain, don't you think so?

**Comment by jamcat22** [comment]
I agree with @mxxcon.

**Comment by NormandLemay** [comment]
I agree with you : @mxxcon , @jamcat22, I will move it.

**Comment by mxxcon** [comment]
What kind of hardware 2fa does it support? I can't read any of that. :/

**Comment by NormandLemay** [comment]
The hardware 2fa is usb keys but, I think his just for the back office. Not for everyone.

**Comment by Carlgo11** [comment]
If the TFA feature isn't for everyone then we won't add it

**Comment by NormandLemay** [comment]
No problem I will remove it

**Comment by Carlgo11** [comment]
Oh that logo is quiet large.
Here's a compressed version of the logo you added:
planethoster

When you've replaced the current logo with that one I think we're done and ready to merge.

**Comment by Carlgo11** [comment]
Nice. Thank you :smiley:

File: _data/hosting.yml [hunk 1]
```diff
@@ -95,6 +95,14 @@ websites:
       tfa: No
       twitter: ovh_support_en
 
+    - name: PlanetHoster
+      url: https://planethoster.net
+      twitter: PlanetHoster
```
Comment (Carlgo11) [inline _data/hosting.yml]: One last thing, could you remove the twitter handle?
Twitter isn't needed for sites already supporting TFA.

**Pull request merged**